packages/server/src/api/routes/tests/deploy.spec.ts

```diff
@@ -0,0 +1,202 @@
+import * as setup from "./utilities"
+import { structures } from "@budibase/backend-core/tests"
+import { features } from "@budibase/backend-core"
+import { createAutomationBuilder } from "../../../automations/tests/utilities/AutomationTestBuilder"
+import { basicTable } from "../../../tests/utilities/structures"
+
+describe("/api/deploy", () => {
+  let config = setup.getConfig()
+
+  afterAll(setup.afterAll)
+
+  beforeAll(async () => {
+    features.testutils.setFeatureFlags("*", {
+      WORKSPACE_APPS: true,
+    })
+    await config.init()
+  })
```

Comment:
You'll want to store the function returned by `setFeatureFlags` and call it in `afterAll` to make sure we're cleaning up after ourselves.